cmd/flag/plan_id_flags.go

```diff
@@ -20,12 +20,12 @@ import (
 )
 
 type PlanIdFlag struct {
-	AcceptedPlanId []string
+	PlanId string
 }
 
 // Add a flag to the command.
 func (f *PlanIdFlag) AddFlags(cmd *cobra.Command) {
-	cmd.Flags().StringSliceVar(&f.AcceptedPlanId, "accepted-plan-id", nil, "Plan ID(s) to accept data from.")
+	cmd.Flags().StringVar(&f.PlanId, "plan-id", "", "Plan ID to stream data for.")
```

Comment:
Do you think it would be worth it to add some bounds checking on the PlanId field?

Reply:
Since a plan ID is a string, I didn't think it would be necessary to validate against anything specific since the API definition doesn't contain any constraints: https://github.com/infostellarinc/stellarstation-api/blob/master/api/src/main/proto/stellarstation/api/v1/stellarstation.proto#L552

Although the plan IDs we see every day are numeric, there's nothing stopping us from generating plan IDs in a different way (with UUIDs or something, for example).